tests/mysql-command/test.yaml

```diff
@@ -0,0 +1,11 @@
+args:
+- -k none
+
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature: "test mysql"
+        
```

Comment:
Could we also test that we have mysql metadata in the signature ?

Reply:
`alert mysql any any -> any any (msg:"test mysql";mysql.command; content:"33030219971120201X"; metadata: mysql command; sid:1;)`,Did you mean this?

Reply:
I think that what he means is something like what is seen in [this test](https://github.com/OISF/suricata-verify/blob/master/tests/pgsql/pgsql-7000-ids/test.yaml#L12-L24), in which we check that the alert event contains app-layer metadata. In the case here, `mysql` app-layer metadata.